SixTrack/sixtrack.s

```diff
@@ -65505,7 +65547,7 @@ c$$$     &           myalphay * cos(phiy))
       function get_dpodx(p,mat_i)          !Claudia
       implicit none
       integer nrmat,nmat,mat,irmat
-      parameter(nmat=12,nrmat=7)
+      parameter(nmat=15,nrmat=12)
```

Comment:
An Astuce block would be nice, instead of repeating the same parameters several times, which requires keeping them in sync -> maintainability problems.